pkg/api/repository.go

```diff
@@ -0,0 +1,46 @@
+package api
+
+import "github.com/content-services/content-sources-backend/pkg/models"
+
+// Repository holds data received from, or returned by, a repositories API request or response
+type Repository struct {
+	UUID                string `json:"uuid" readonly:"true"`
+	Name                string `json:"name"`
+	URL                 string `json:"url"`                                //URL of the remote yum repository
+	DistributionVersion string `json:"distribution_version" example:"7"`   //Version to restrict client usage to
+	DistributionArch    string `json:"distribution_arch" example:"x86_64"` //Architecture to restrict client usage to
+	AccountID           string `json:"account_id" readonly:"true"`         //Account ID of the owner
+	OrgID               string `json:"org_id" readonly:"true"`             //Organization ID of the owner
+}
+
+// CreateRepository holds data received from request to create repository
+type CreateRepository struct {
+	UUID                string `json:"uuid" readonly:"true" swaggerignore:"true"`
+	Name                string `json:"name"`
+	URL                 string `json:"url"`                                             //URL of the remote yum repository
+	DistributionVersion string `json:"distribution_version" example:"7"`                //Version to restrict client usage to
+	DistributionArch    string `json:"distribution_arch" example:"x86_64"`              //Architecture to restrict client usage to
+	AccountID           string `json:"account_id" readonly:"true" swaggerignore:"true"` //Account ID of the owner
+	OrgID               string `json:"org_id" readonly:"true" swaggerignore:"true"`     //Organization ID of the owner
```

Comment:
we don't want AccountID and OrgId on this struct, as its pulled from the headers and not passed by the user

Reply:
ohh i see you've got swaggerignore on it  👍🏻